docs/developer/tutorials/jmx/tools.md

```diff
@@ -0,0 +1,55 @@
+# JMX Tools
```

Comment:
Looks like this will lead to 404 on the tutorials page, you need to specify the subpage `tools.md`

https://github.com/DataDog/integrations-core/blob/da2e5aa5a340ff10a9090baf0086ff96c59681b2/mkdocs.yml#L62

Reply:
@ChristineTChen Thx. Updated.